packages/hdwallet-provider/src/constructor/getOptions.ts

```diff
@@ -2,14 +2,11 @@ import type { MnemonicPhrase, PrivateKey } from "./types";
 import type { ConstructorArguments } from "./ConstructorArguments";
 import type * as Constructor from "./Constructor";
 import type * as LegacyConstructor from "./LegacyConstructor";
-import { validateMnemonic } from "ethereum-cryptography/bip39";
-import { wordlist } from "ethereum-cryptography/bip39/wordlists/english";
 
 // check that the first argument is a mnemonic phrase
 const isMnemonicPhrase = (
   credentials: LegacyConstructor.Credentials
-): credentials is MnemonicPhrase =>
-  typeof credentials === "string" && validateMnemonic(credentials, wordlist);
+): credentials is MnemonicPhrase => typeof credentials === "string";
```

Comment:
Why was the validation removed here?  I assume this is related to what you said on the PR comments but the explanation wasn't really clear... could you explain in more detail?

Reply:
Also, I think this function should be renamed... or actually, removed.  The name is misleading now because it's no longer performing validation.  Looking at the changes below, this makes sense, but the function name is misleading, so I think the function should be done away with.

Reply:
Consider the case where I use "I am having fun" as my mnemonic. In that case this check will say "hey, that is not a valid mnemonic and therefore is not a mnemonic - so it must be a private key". Then it returns the `signingAuthority` stuff in an incorrect form: namely like
```
{ privateKey: "I am having fun" }
```
It seems to me that here we just want to figure out whether what they entered is "mnemonic-like". Later we do this validation again anyways.

Reply:
For reference, we validate the mnemonic again [here](https://github.com/trufflesuite/truffle/blob/develop/packages/hdwallet-provider/src/index.ts#L312).

Reply:
Yeah if you add the spaces check that makes sense!